README.md

```diff
@@ -1,97 +1,174 @@
-# HIVE FIVE
+<h2 align="center">
+    <a href="https://hivefive.online" target="blank_">
+        <img height="100" alt="HiveFive" src="docs/resources/colored_logo.svg" />
+    </a>
+    <br>
+    HiveFive: a beekeeper's companion
+</h2>
 
-HIVE FIVE app is composed of:
+[![Symfony version][symfony_version_img]][symfo_url]
+[![React version][react_version_img]][react_url]
+[![Wiki][repo_wiki_img]][repo_wiki_url]
+[![License][repo_license_img]][repo_license_url]
 
-- A [front application](front/README.md), as an IHM developed with React
-- A [back application](./back/README.md), as a web API developed with Symfony
+Manage your **beekeeping** activities with **HiveFive**. Collect and analyze data about your hives, apiaries, and more. Get **insights** about your apiaries and hives, and share your data with other beekeepers.
 
-## Requirements
+- [🐝 Features](#-features)
+  - [📊 Data collection](#-data-collection)
+  - [📈 Data analysis](#-data-analysis)
+  - [📱 Mobile-friendly](#-mobile-friendly)
+- [📁 Root project structure](#-root-project-structure)
+- [⚡️ Quick start](#️-quick-start)
+  - [🐳 Docker-way to quick start](#-docker-way-to-quick-start)
+- [📚 Documentation](#-documentation)
+- [⚙️ Settings \& Commands](#️-settings--commands)
+- [🤝 Build with us!](#-build-with-us)
+- [⚠️ License](#️-license)
 
-- [Node.js](https://nodejs.org/en/) (>= 18)
-- [Docker](https://www.docker.com/)
-- [PostgreSQL](https://www.postgresql.org/) (>= 13) (Optional)
-- [PHP](https://www.php.net/) (>= 8.2) (Optional)
-- [Symfony CLI](https://symfony.com/download) (Optional)
+## 🐝 Features
 
-## Quickstart
+### 📊 Data collection
 
-### Development tools
+- **HiveFive** allows you to collect data about your hives and apiaries. You can add as many hives and apiaries as you want, and add data about them. You can also add **inspections** to your hives, and add data about them.
 
-You can install the development environment with the following command:
+[![HiveFive - Apiary collection](docs/resources/apiary_list.png)](https://hivefive.online)
+
+### 📈 Data analysis
+
+(Coming soon)
+
+### 📱 Mobile-friendly
+
+**HiveFive** is mobile-friendly. You can use it on your phone, tablet, or computer. The interface is responsive and adapts to your device.
+
+[![HiveFive - Mobile-friendly](docs/resources/mobile_friendly.png)](https://hivefive.online)
```

Comment:
![Capture d’écran 2024-06-17 à 16 41 23](https://github.com/Hive-Five-project/hive-five/assets/113915173/fa7777dd-83d5-4105-98bf-ad48235dda64)

is normal ?

Reply:
I forgot! :(